src/System/TimedScopes/ITimedScopeStackManager.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.Omex.System.TimedScopes
+{
+	/// <summary>
+	/// Interface for handling active scopes stack
+	/// </summary>
+	public interface ITimedScopeStackManager
+	{
+		/// <summary>
+		/// Get a stack of active scopes, creating a new stack if one does not exist
+		/// </summary>
+		/// <returns>stack of scopes</returns>
+		TimedScopeStack GetTimedScopeStack();
```

Comment:
>GetTimedScopeStack [](start = 18, length = 18)

Can this be replaced with 





public interface ITimedScopeStackManager
 {
  
  TimedScopeStack {get; set; };
}

?

Reply:
I can change this, however, if I am not mistaken it was @AndreyTretyak suggestion to have separate methods instead of property.

Reply:
Can you please ask him what the reason for that was?

---
In reply to: [314391362](https://github.com/microsoft/Omex/pull/96#discussion_r314391362) [](ancestors = 314391362)

Reply:
Replaced